src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -74,25 +80,32 @@ def __call__(
         latents = torch.randn(
             (batch_size, self.unet.in_channels, self.unet.sample_size, self.unet.sample_size),
             generator=generator,
+            device=torch_device,
```

Comment:
This is the way stable diffusion does it - I don't think it's the best way as it doesn't allow CPU and GPU to have the same random numbers, but here I think it's more important that we're 1-to-1 the same as stable diffusion

Reply:
Agreed!